targets/nordic_pca10056-blehci-usb/syscfg.yml

```diff
@@ -0,0 +1,7 @@
+syscfg.vals:
+    BLE_HCI_TRANSPORT: usb
+    BLE_PUBLIC_DEV_ADDR: "(uint8_t[6]){0x81, 0x19, 0x1A, 0xC0, 0xCA, 0xC0}"
```

Comment:
I'm not sure if we should hardcode public addr in samples, and BlueZ is able to handle controllers without one just fine

Reply:
@kasjer please just remove BLE_PUBLIC_DEV_ADDR

Reply:
done